src/ImageSharp/Formats/Jpeg/Components/FloatingPointDCT.Intrinsic.cs

```diff
@@ -20,34 +20,34 @@ private static void FDCT8x8_Avx(ref Block8x8F block)
         FDCT8x8_1D_Avx(ref block);
 
         // Second pass - process rows
-        block.TransposeInplace();
+        block.TransposeInPlace();
         FDCT8x8_1D_Avx(ref block);
 
         // Applies 1D floating point FDCT inplace
         static void FDCT8x8_1D_Avx(ref Block8x8F block)
         {
-            Vector256<float> tmp0 = Avx.Add(block.V0, block.V7);
-            Vector256<float> tmp7 = Avx.Subtract(block.V0, block.V7);
-            Vector256<float> tmp1 = Avx.Add(block.V1, block.V6);
-            Vector256<float> tmp6 = Avx.Subtract(block.V1, block.V6);
-            Vector256<float> tmp2 = Avx.Add(block.V2, block.V5);
-            Vector256<float> tmp5 = Avx.Subtract(block.V2, block.V5);
-            Vector256<float> tmp3 = Avx.Add(block.V3, block.V4);
-            Vector256<float> tmp4 = Avx.Subtract(block.V3, block.V4);
+            Vector256<float> tmp0 = Avx.Add(block.V256_0, block.V256_7);
+            Vector256<float> tmp7 = Avx.Subtract(block.V256_0, block.V256_7);
+            Vector256<float> tmp1 = Avx.Add(block.V256_1, block.V256_6);
+            Vector256<float> tmp6 = Avx.Subtract(block.V256_1, block.V256_6);
+            Vector256<float> tmp2 = Avx.Add(block.V256_2, block.V256_5);
+            Vector256<float> tmp5 = Avx.Subtract(block.V256_2, block.V256_5);
+            Vector256<float> tmp3 = Avx.Add(block.V256_3, block.V256_4);
+            Vector256<float> tmp4 = Avx.Subtract(block.V256_3, block.V256_4);
```

Comment:
nit: These could use `x + y` and `x - y`. Similar for other arithmetic operations in the method